README.md

```diff
@@ -35,4 +45,15 @@ module.exports = {
 };
 ```
 
+If you don't need Vue you can use base configuration:
```

Comment:
How about adding example of how to set the `overrides` if your client code is not in `client` directory (e.g. in case of a vue lib with code in `src` or somewhere else)

Reply:
While I agree I fail to see how is that relevant in terms of this PR? I mean we should address it separately because it is actually applicable to both config variants? :thinking:

Reply:
@vladimyr I agree. Tracking it via #5